io/include/traccc/io/data_format.hpp

```diff
@@ -9,9 +9,7 @@
 
 namespace traccc {
 
-enum data_format : int {
-    csv = 0,
-    binary = 1,
-};
+/// Format for an input or output file
+enum data_format : int { csv = 0, binary = 1, json = 2 };
```

Comment:
Could you put a trailing comma behind the 2 so `clang-format` formats this properly?

Reply:
:+1: